src/GafferUI/StandardConnectionGadget.cpp

```diff
@@ -478,7 +504,7 @@ void StandardConnectionGadget::updateDotPreviewLocation( const ButtonEvent &even
 
 bool StandardConnectionGadget::keyPressed( const KeyEvent &event )
 {
-	if( event.modifiers & ButtonEvent::Control )
+	if( event.modifiers & ButtonEvent::Control && srcNodule() && dstNodule() )
```

Comment:
Note that `dstNodule()` is actually [guaranteed to be non-null](https://github.com/GafferHQ/gaffer/blob/master/src/GafferUI/ConnectionGadget.cpp#L92). I think you've picked up this "belt and braces" approach from code I wrote earlier in the file, but it's better to omit unnecessary guards because they spread confusion and mask bugs.